templates/demo-store/app/components/SortFilter.tsx

```diff
@@ -279,7 +256,7 @@ function PriceRangeFilter({max, min}: {max?: number; min?: number}) {
         <input
           name="maxPrice"
           className="text-black"
-          type="text"
+          type="number"
```

Comment:
Not sure why the max price was a `text` type before, let me know if there could be a special reason for this